docs/_quarto.yml

```diff
@@ -367,9 +367,63 @@ quartodoc:
             name: Temporal expressions
             desc: Dates, times, timestamps and intervals.
           contents:
-            - TimestampValue
-            - DateValue
-            - TimeValue
+            - name: TimestampValue
+              include_inherited: true
+              members:
+                - add
+                - radd
+                - sub
+                - between
+                - bucket
+                - date
+                - day
+                - day_of_week
+                - day_of_year
+                - delta
+                - epoch_seconds
+                - hour
+                - microsecond
+                - millisecond
+                - minute
+                - month
+                - quarter
+                - second
+                - strftime
+                - time
+                - truncate
+                - week_of_year
+                - year
+            - name: DateValue
+              include_inherited: true
+              members:
+                - add
+                - radd
+                - sub
+                - day
+                - day_of_week
+                - day_of_year
+                - epoch_seconds
+                - month
+                - quarter
+                - strftime
+                - truncate
+                - week_of_year
+                - year
+            - name: TimeValue
+              include_inherited: true
```

Comment:
Do we need this? If we're explicitly listing the inherited members it doesn't seem like we should.

Reply:
I just tested this locally, and you are right. If we list them we don't need the `include_inherited: true`
```suggestion
```